src/style-spec/reference/v8.json

```
@@ -2816,7 +2825,7 @@
         }
       },
       "format": {
-        "doc": "Returns `formatted` text containing annotations for use in mixed-format `text-field` entries. For a `text-field` entries of a string type, following option object's properties are supported: If set, the `text-font` value overrides the font specified by the root layout properties. If set, the `font-scale` value specifies a scaling factor relative to the `text-size` specified in the root layout properties. If set, the `text-color` value overrides the color specified by the root paint properties for this layer.",
+        "doc": "Returns a `formatted` string for use in the `text-field` property. The input may contain a plain `string`, an `image` provided through the [`'image'`](#types-image) expression, or a combination  of the two. Supported styling options:\n- `\"text-font\"`: Overrides the font stack specified by the root layout property.\n- `\"text-color\"`: Overrides the color specified by the root paint property.\n- `\"font-scale\"`: Applies the specified a scaling factor relative to the `text-size` specified by the root layout property.",
```

Removing the `mixed-format` makes it harder to understand the purpose of this expression type and when to use it. It's also not clear from the description how to pair the sections with their override options.

```suggestion
        "doc": "Returns a `formatted` string for displaying mixed-format text in the `text-field` property. The input may contain a string literal or expression, or an  [`'image'`](#types-image) expression, or a combination  of the two. Strings can be followed with a style override object that supports the following properties:\n- `\"text-font\"`: Overrides the font stack specified by the root layout property.\n- `\"text-color\"`: Overrides the color specified by the root paint property.\n- `\"font-scale\"`: Applies the specified number as a scaling factor relative to the `text-size` specified by the root layout property.",
```